src/main/scala/com/softwaremill/realworld/articles/ArticlesRepository.scala

```diff
@@ -121,6 +115,10 @@ class ArticlesRepository(quill: SqliteZioJdbcContext[SnakeCase], dataSource: Dat
         _.authorId -> lift(userId)
       )
   ).unit
+    .mapError {
+      case e: SQLiteException if e.getResultCode == SQLiteErrorCode.SQLITE_CONSTRAINT_PRIMARYKEY =>
```

Comment:
Not all cases are handled here. Please add fallback:
`e => e`
Otherwise an exception would be thrown when exception can't be matched

Reply:
We should enable match exhaustivity warnings / errors then :)

Reply:
It was fixed in another PR ;)
https://github.com/softwaremill/realworld-tapir-zio/pull/24